crates/network/src/any/mod.rs

```diff
@@ -0,0 +1,35 @@
+use alloy_rpc_types::{Extra, Header, Transaction, TransactionReceipt, TransactionRequest};
+
+use crate::{Network, ReceiptResponse};
+
+mod builder;
+
+/// Types for a catch-all network.
```

Comment:
please expand the docs a bit on when you would use this (and also discourage its usage), and what it does

you would only use it if you do not know the network type at compile time